lib/Convos.pm

```diff
@@ -129,9 +113,17 @@ sub _config {
     ||= $config->{forced_irc_server} || $ENV{CONVOS_DEFAULT_SERVER} || 'localhost';
   $config->{invite_code} ||= $ENV{CONVOS_INVITE_CODE} // $self->_generate_invite_code;
   $config->{organization_name} ||= $ENV{CONVOS_ORGANIZATION_NAME} || 'Nordaaker';
-  $config->{plugins} ||= {};
-  $config->{plugins}{$_} = $config for split /,/, +($ENV{CONVOS_PLUGINS} // '');
-  $config->{secure_cookies} ||= $ENV{CONVOS_SECURE_COOKIES} || 0;
+  $config->{secure_cookies}    ||= $ENV{CONVOS_SECURE_COOKIES}    || 0;
+
+  # public settings
+  $config->{settings} = {
+    contact           => $config->{contact},
+    default_server    => $config->{default_server},
+    forced_irc_server => $config->{forced_irc_server} ? true : false,
+    invite_code       => $config->{invite_code} ? true : false,
+    organization_name => $config->{organization_name},
+  };
```

Comment:
It's much easier for plugins to change the public settings, when it's a variable instead of a helper.